README.md

```diff
@@ -192,11 +192,11 @@ CONTAINER ID        IMAGE               COMMAND                  CREATED
 7d7b47ba4c72        nginx-agent       "/entrypoint.sh"         22 minutes ago      Exited (137) 19 seconds ago                       mynginx1
 ```
 
-## 3.0 Adding the Controller Agent during container run
+### 2.3. Adding the Controller Agent during container run
 
 An alternate way to handle the Controller Agent within containers is to include the necessary Controller Agent commands in the run command for the container. This way, you don't have to build the Controller Agent into your container before running.
```

Comment:
What is happening here is the customer is beginning with an existing image and adding an additional layer that contains only the Controller Agent.
This is great for PoCs.  We recommend incorporating agent into the regular image build process for production.